### PR TITLE
Clarify message on broken links

### PR DESCRIPTION
**Pull request by gtback**
For new people and those less familiar with the broken links messages,
it's not always clear which page has the broken link, and which page the
link is pointing to. Hopefully this clears it up!

### CODE REVIEW

**Review by bmorelli25 — APPROVED** [review]
Great improvement! 🕺💃 

Why is broken in parenthesis?

File: lib/ES/LinkCheck.pm [hunk 1]
```diff
@@ -96,7 +96,7 @@ sub report {
 
     my @error = "Bad cross-document links:";
     for my $file ( sort keys %$bad ) {
-        push @error, "  $file:";
+        push @error, "  $file contains (broken) links(s) to:";
```
Comment (lockewritesdocs) [inline lib/ES/LinkCheck.pm:99]:
```suggestion
        push @error, "  $file contains broken links to:";
```

**Review by lockewritesdocs — APPROVED** [review]
Left a minor change, but LGTM otherwise. Thanks @gtback!

**Review by jrodewig — APPROVED** [review]
LGTM with the mentioned changes.

**Pull request merged**